torch/csrc/jit/register_special_ops.cpp

```diff
@@ -94,7 +95,27 @@ RegisterOperators reg({
             push(stack, at::infer_size(a, b));
             return 0;
           };
-        })
+        }),
+    Operator(
+      "aten::_no_grad_embedding_renorm_(Tensor weight, Tensor input, float max_norm, float norm_type) -> Tensor",
```

Comment:
Do you know if FloatType uses float or double internally?

Reply:
Double